int16 up_down
int16 left_right
int16 far_near
